app/policies/employees/user_policy.rb

```diff
@@ -3,6 +3,8 @@
 module Employees
   # Check current User credentials
   class UserPolicy < ApplicationPolicy
+    include Credentials
```

Comment:
why we use this? as all you've moved is related to user

Reply:
the included module provide permission checker
this module going to use in other policies like invitation, locking, teams...
I propose to rename Credentials module to PermissionChecker

Reply:
let's rename